harness/determined/keras/_tf_keras_trial.py

```diff
@@ -26,49 +29,52 @@
 IMPOSSIBLY_LARGE_EPOCHS = sys.maxsize
 
 
-def load_optimizer_weights(model: Model, load_path: pathlib.Path) -> None:
+def load_optimizer_weights(
+    model: Model, h5group: Any, optimizer: Optional[tf.keras.optimizers.Optimizer]
```

Comment:
I don't understand `optimizer` would ever be `None`?  Also, can we just handle that case externally to this function in that one legacy code case?

Reply:
Yeah, can we just feed `model.optimizer` into this function for that case?

Like:
```diff
            if "optimizer_weights" in h5file:
-               load_optimizer_weights(self.model, h5file["optimizer_weights"], None)
+               load_optimizer_weights(self.model, h5file["optimizer_weights"], model.optimizer)
```